vtStorAtaProtocol/Platform/Linux/ProtocolAtaPassThrough.h

```diff
@@ -34,7 +34,7 @@ class VT_STOR_ATA_PROTOCOL_API cAtaPassThrough : public cProtocolInterface
 {
 
 public:
-    //virtual eErrorCode IssueCommand( const DeviceHandle& Handle, std::shared_ptr<const cBufferInterface> Essense, std::shared_ptr<cBufferInterface> DataBuffer ) override;
+    virtual eErrorCode IssueCommand( const DeviceHandle& Handle, std::shared_ptr<const cBufferInterface> Essense, std::shared_ptr<cBufferInterface> DataBuffer ) override;

```

Comment:
I know it's Linux code. But the thing is it's implemented for another task. And it isn't completed. Code to trunk MUST be completed and for specific task.